### PR TITLE
Give HFR gas monitoring bar a decreasement/increasement rate, slight tweak to HFR tgui and increase HFR effciency due to recent process changes

### PR DESCRIPTION
**Pull request by warface1234455**
thank you to @SapphicOverload for helping me in few things
# Document the changes in your pull request
HFR gas monitoring bar has a decreasement/increasement rate rate for how much gas being removed per tick
Moderator now list all sticky gases
increase HFR effciency due to recent process change that decreased HFR performance

# Why is this good for the game?
You get to know how much gas is being removed from the HFR, pretty convenient

# Testing

crate

# Wiki Documentation
no wiki

# Changelog

:cl:  
rscadd: Give HFR gas monitoring bar a decreasement/increasement rate rate
tweak: Moderator now list all sticky gases
tweak: increase HFR effciency due to recent process change that decreased HFR performance
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
"Ready for review" 5 more comits

**Comment by iloveloopers** [comment]
Average coder

**Comment by ShoziOffical** [comment]
> "Ready for review" 5 more comits

slight tweak lol

File: code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm [hunk 1]
```diff
@@ -269,30 +280,42 @@
...
 				internal_output.adjust_moles(GAS_BZ, scaled_production * 1.8)
-				moderator_internal.adjust_moles(GAS_PLASMA, -min(moderator_internal.get_moles(GAS_PLASMA), scaled_production * 1.75))
+				var/remove_amount = min(moderator_internal.get_moles(GAS_PLASMA), scaled_production * 1.75)
+				moderator_internal.adjust_moles(GAS_PLASMA, -remove_amount)
+				delta_mod_removed_list[GAS_PLASMA] = -remove_amount
 			if(moderator_list[GAS_PLUONIUM] > 20)
 				radiation *= 1.55
-				heat_output *= 1.025
-				moderator_internal.adjust_moles(GAS_PLUONIUM, -min(moderator_internal.get_moles(GAS_PLUONIUM), scaled_production * 1.35))
+				heat_output *= 1.
```
Comment (iamthesilly) [inline code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm]:
```suggestion
				heat_output *= 1.025
```

File: code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm [hunk 2]
```diff
@@ -319,18 +346,24 @@
 			if(moderator_list[GAS_HEALIUM] > 100)
 				if(critical_threshold_proximity > 400)
 					critical_threshold_proximity = max(critical_threshold_proximity - (moderator_list[GAS_HEALIUM] / 100 * delta_time ), 0)
-					moderator_internal.adjust_moles(GAS_HEALIUM, -min(moderator_internal.get_moles(GAS_HEALIUM), scaled_production * 20))
+					var/remove_amount = min(moderator_internal.get_moles(GAS_HEALIUM), scaled_production * 20)
+					moderator_internal.adjust_moles(GAS_HEALIUM, -remove_amount)
+					delta_mod_removed_list[GAS_PLASMA] = -remove_amount
```
Comment (iamthesilly) [inline code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm]:
```suggestion
					delta_mod_removed_list[GAS_HEALIUM] = -remove_amount
```

**Review by SapphicOverload — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm]: oh shi

File: code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm (hunk 2 above)
Comment (ShoziOffical) [reply code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm]: i swear i rechecked

**Comment by ToasterBiome** [comment]
Reopen when done

**Pull request closed without merging**